src/otrs-popup/otrs-popup.controller.js

```diff
@@ -301,6 +305,10 @@ angular.module("ovh-angular-otrs").controller("OtrsPopupCtrl", function ($q, $ro
             self.categories = results.supportSchema.models["support.TicketProductEnum"].enum;
             self.requests = results.supportSchema.models["support.TicketCategoryEnum"].enum;
 
+            if (_.get(results.me, "ovhSubsidiary").toLowerCase() === "fr") {
```

Comment:
define default value to string : `if (_.get(results.me, "ovhSubsidiary", "").toLowerCase() === "fr") {` to avoid `Cannot read property 'toLowerCase' of undefined`